dsm/resource_aws_sobject.go

```diff
@@ -119,7 +120,9 @@ func resourceAWSSobject() *schema.Resource {
 				"   * `interval_days`: Rotate the key for every given number of days.\n" +
 				"   * `interval_months`: Rotate the key for every given number of months.\n" +
 				"   * `effective_at`: Start of the rotation policy time.\n" +
-				"   * **Note:** Either interval_days or interval_months should be given, but not both.",
+				"   * `deactivate_rotated_key`: This is not supported. Please provide `false` to avoid the changes detected during terraform plan.\n" +
```

Comment:
Is the need to provide `false` a quirk of terraform? Seems like we should be able to default that value somehow (since the zero value of a boolean in Go is false).

Reply:
Removed deactivate_rotated_key and handled in tf state(Read function.).